homeassistant/components/lamarzocco/config_flow.py

```diff
@@ -56,7 +56,7 @@ async def async_step_user(
                 _LOGGER.debug("Server rejected login credentials")
                 errors["base"] = "invalid_auth"
             except RequestNotSuccessful as exc:
-                _LOGGER.exception("Error connecting to server: %s", str(exc))
+                _LOGGER.error("Error connecting to server: %s", str(exc))
```

Comment:
Exceptions have a magic string method so we don't need to copy to string.
```suggestion
                _LOGGER.error("Error connecting to server: %s", exc)
```